app/person-escort-record/middleware.js

```diff
@@ -14,12 +14,16 @@ function setFrameworkSection(frameworkSection) {
   }
 }
 
-async function setPersonEscortRecord(req, res, next, recordId) {
+async function setPersonEscortRecord(req, res, next) {
+  const recordId =
+    req.params?.personEscortRecordId || req?.personEscortRecord?.id
```

Comment:
Not following this logic. The only place I can see where do we get `req.personEscortRecord` is later in this method

Reply:
This allows us to support the app both nested and on its own. So if the ID is set in the request params it will use that ID to get the record, otherwise it will look to see if an ID has been set within another middleware (currently we do this in the move app so that the router can be nested a whole).

At the moment it's not hugely efficient as we need to make another call to get the PER with all the responses. I didn't want to add the responses to the default `include` of `move` as it would've meant asking for them for each item in the GET all request we make as we haven't split our GET one vs GET all calls yet.